src/locales/en/translation.json

```diff
@@ -487,8 +487,8 @@
           "previousAddress": "Previous address"
         },
         "updateUserDetailsInformation": {
-          "updateDetailsLabel": "How to update your details",
-          "updateDetailsHtml": "You can <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"{{url}}\">contact the GOV.UK One Login team (opens in new tab)</a> to update your details."
+          "updateDetailsLabel": "If your details are out of date",
+          "updateDetailsHtml": "<p>Your proof of identity is still valid even if your details are out of date.</p><p>You do not need to update your details to continue to the service you need to use. When you continue, GOV.UK One Login will share your current details with the service.</p><p>If you want to update your details before they are shared with the service, you'll need to:</p><ul><li>delete your details</li><li>prove your identity again</li></ul><a class=\"govuk-link\" href=\"/ipv/journey/reset-identity\" rel=\"noopener noreferrer\">Prove your identity again with new details</a>."
```

Comment:
Doesn't this need to be a post request/form submit to correctly handle the reset-identity event?

Reply:
That was my original thought, but then that involves turning the anchor tag into a button (since you can't POST using an anchor tag) and then having to have custom styling to make the button look like a hyperlink.

This option makes a get request, which is picked up using the existing handler in middleware and passed straight to the backend as an updateJourneyState request, which feels a lot cleaner (albeit not quite in the spirit of semantic html)